torch/nn/script/activation.py

```diff
@@ -0,0 +1,840 @@
+import warnings
```

Comment:
man.... I'm not so sure about copy pasting all of them. Can you define these programmatically so they are always in sync?

Reply:
It was copied and pasted originally, but made bunch of changes in each of class. It's pretty hard to it in sync with torch.nn as JIT/Script don't support full set of python, we always need to make some customization to make it into the subset of python syntax so that the JIT/Script mode will work properly. I will try to make it as generalized and possible.

Reply:
What is the summary of changes you had to do? I wonder whether we can modify the original implementation accordingly.

At least we should share the doc string - it should be doable with python surgery of setting __doc__

Reply:
@dzhulgakov I will soon add a summary of strategy we discussed so far. As of now, as the shortage in supporting multiple features with script, we will make a copy of the nn modules and functional interfaces to do an initial support. After we resolved the missing features in script, we will directly modify the original implementation. I will update the PR to make use of setting doc.